src/torchmetrics/functional/classification/specificity_sensitivity.py

```diff
@@ -403,7 +403,7 @@ def multilabel_specificity_at_sensitivity(
     return _multilabel_specificity_at_sensitivity_compute(state, num_labels, thresholds, ignore_index, min_sensitivity)
 
 
-def specicity_at_sensitivity(
+def specificity_at_sensitivity(
```

Comment:
this is rename public function, so someone was using it in his project; this change will break his code

Reply:
I see. Then do you think this rename operation is still worth to do, or is it better to keep it as "specicity"?

Reply:
cc: @lantiga

Reply:
With adding come compatibility path and deprecation warning, all shall be smooth...

Reply:
Yep let's add the new one, keep the old one around and add the deprecation warning.

Reply:
Thanks for the fix @oguz-hanoglu

Reply:
@oguz-hanoglu could you pls follow examples with deprecations and set it for removal in 2.0

Reply:
@Borda I added the old method back and added a deprecation warning.